expansion/src/main/java/no/unit/nva/expansion/model/ExpandedResource.java

```diff
@@ -82,12 +81,14 @@ public String getType() {
         return this.allFields;
     }
 
-    public List<URI> getPublicationContextUris() {
-        ObjectNode docAsObjectNode = objectMapper.convertValue(this.allFields, ObjectNode.class);
-        return getPublicationContextUris(docAsObjectNode);
+    @Override
+    @JsonIgnore
+    public SortableIdentifier fetchIdentifier() {
+        return SortableIdentifier.fromUri(fetchId());
     }
 
-    public URI getId() {
+
+    public URI fetchId() {
```

Comment:
method for getting the id without duplicating the Id field that is injected in the contained json object during the creation of an ExpandedResource instance.